e2e/suites/provider/cases/aws/common.go

```diff
@@ -31,6 +31,11 @@ const (
 	WithMountedIRSA                     = "with mounted IRSA"
 	StaticCredentialsSecretName         = "provider-secret"
 	StaticReferentCredentialsSecretName = "referent-provider-secret"
+
+	IAMRoleExternalID  = "arn:aws:iam::783882199045:role/eso-e2e-external-id"
+	IAMRoleSessionTags = "arn:aws:iam::783882199045:role/eso-e2e-session-tags"
```

Comment:
These roles are set up here and allow access to secrets that have specific tags:

https://github.com/external-secrets/infrastructure/blob/main/tf/aws/e2e_external_id.tf
https://github.com/external-secrets/infrastructure/blob/main/tf/aws/e2e_session_tags.tf